app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -53,6 +53,13 @@ def find_by_installable_errata(_key, operator, value)
             { :conditions => "#{::Host::Managed.table_name}.id IN (#{facets.pluck(:host_id).join(',')})" }
           end
         end
+
+        def in_content_view_environment(content_view = nil, lifecycle_environment = nil)
```

Comment:
I feel like this should maybe just return the relation or at least require one of these when calling it? Seems strange to allow both to be nil when its specifically 'in_content_view_environment' and not in content view and/or environment?